### PR TITLE
Additional ini updates

### PR DESCRIPTION
**Pull request by RingoStarr39**

### CODE REVIEW

File: ini/rthun2.ini [hunk 1]
```diff
@@ -1,41 +1,44 @@
...
+length   = 0x80000
+position = 0x100000
+
+[wave.1]
+filename = voi2.bin
+length   = 0x80000
+position = 0x300000
+
+[playlist]
+04d = ATTRACT
```
Comment (superctr) [inline ini/rthun2.ini]: Not sure what's the point of capitalizing everything here. I prefer to keep mixed case if the OST track names are all uppercase

File: ini/adillor.ini [hunk 2]
```diff
@@ -1,41 +1,41 @@
...
+position = 0x0
+
+[wave.1]
+filename = am1waveb.1l
+length   = 0x400000
+position = 0x800000
+
+[playlist]
+; http://vgmdb.net/album/3065
+8d = Armadillo And His Friends [Attract 1]
```
Comment (superctr) [inline ini/adillor.ini:24]: source on the translated track titles? The original ones were taken from vgmdb as mentioned in the file.

**Review by superctr — COMMENTED** [review]
I'm OK with most of the changes. If you change the RT2 track titles back to mixed case I will merge this PR as it is.

File: ini/rthun2.ini (hunk 1 above)
Comment (RingoStarr39) [reply ini/rthun2.ini]: Yeah, I was just going off of the soundtrack releases. I can change it back.

File: ini/adillor.ini (hunk 2 above)
Comment (RingoStarr39) [reply ini/adillor.ini:24]: The previous translations were a bit...off. So I re-translated them myself using DeepL and other sources.

**Comment by RingoStarr39** [comment]
I can make the changes you requested. Would it be easier to close this PR and submit a new one?

**Comment by superctr** [comment]
If you push a commit to the same branch you used for this PR, it will also
update the pull request. So it shouldn't be necessary to make a new PR.

Den tis 4 jan. 2022 03:02RingoStarr39 ***@***.***> skrev:

**Pull request merged**